### PR TITLE
Introduces Environment Variables.

### PR DESCRIPTION
**Pull request by dadamsretrorabbit**
Introduces Environment Variables

### CODE REVIEW

File: waterlog/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_API_URL=https://api.iot.retrotest.co.za
```
Comment (FredJEhlers) [inline waterlog/.env:1]: I am sure these go in the gitignore

Comment (FredJEhlers) [reply waterlog/.env:1]: I mean these files

**Review by FredJEhlers — CHANGES_REQUESTED** [review]

File: waterlog/.env (hunk 1 above)
Comment (FredJEhlers) [reply waterlog/.env:1]: Unless you are not going to use these env files to not store passwords? in which case seems like its fine and I will resolve

Comment (aeTherZA) [reply waterlog/.env]: These will never be used to store passwords. Only URL to specify the backend API

Comment (dadamsretrorabbit) [reply waterlog/.env:1]: This will never be used to store passwords.

**Review by FredJEhlers — APPROVED** [review]

**Pull request merged**